pkg/cmd/grafana-server/main.go

```diff
@@ -51,10 +51,11 @@ func (e exitWithCode) Error() string {
 
 func main() {
 	var (
-		configFile = flag.String("config", "", "path to config file")
-		homePath   = flag.String("homepath", "", "path to grafana install/home path, defaults to working directory")
-		pidFile    = flag.String("pidfile", "", "path to pid file")
-		packaging  = flag.String("packaging", "unknown", "describes the way Grafana was installed")
+		configFile  = flag.String("config", "", "path to config file")
+		homePath    = flag.String("homepath", "", "path to grafana install/home path, defaults to working directory")
+		pidFile     = flag.String("pidfile", "", "path to pid file")
+		packaging   = flag.String("packaging", "unknown", "describes the way Grafana was installed")
```

Comment:
Oh, I didn't know there was a `packaging` flag. Would it be OK to use this field for GitLab embedding like `-packaging="GitLab Omnibus"`?

Reply:
Ahh, nevermind, `validPackaging()` restricts it to specific strings.

Reply:
Gitlab is not a valid package https://github.com/grafana/grafana/pull/31039/files#diff-828368acf3145252e4e01950c3f59e16cc7db5b40df7cb6da9436c6d77e44112R174

Reply:
Want to add `embedded` to the `validTypes` list?

Reply:
I've instructed GitLab to use the packaging flag historically. Maybe something did change in regards to this